04/README.md

```diff
@@ -0,0 +1,77 @@
+# Lesson 4
+
+## Content
+- Collections
+  - List
+  - Set
+  - Map
+- Collection/MutableCollection
+- Basic write operations
+- Common operations
+  - Transformations
+  - Filtering
+  - Grouping 
+  ...
+
+## Useful links
+- [Slack channel](https://appport-academy.slack.com/) (invite can be requested via [event link](https://appport.cz/event/appport-academy/) or [Meetup group](https://www.meetup.com/Prague-Mobile-Development-Meetup/events/269765161/))
+- [Github](https://github.com/Concur-Kotlin-Academy/academy2020)
+- Kotlin manual: [link](https://kotlinlang.org/docs/reference/collections-overview.html)
+
+## Tasks
+### Project creation
+1. open IntelliJ idea
+1. select create new project
+1. choose Kotlin/JVM
+1. name the project "lesson-02"
+1. create new Kotlin file in the `src` folder with name `Main`
+1. add your first Kotlin code
+```kotlin
+fun main(){
+    println("Hello World")
+}
+```
```

Comment:
Empty line before this code and indent it with 4 spaces, otherwise the next bullet point restarts at 1.